protocol/nats_jetstream/v2/protocol.go

```diff
@@ -67,6 +86,37 @@ func NewProtocolFromConn(conn *nats.Conn, stream, sendSubject, receiveSubject st
 	return p, nil
 }
 
+func NewProtocolFromConnV2(conn *nats.Conn, stream, sendSubject string, jsOpts []jetstream.JetStreamOpt, opts ...ProtocolOption) (*Protocol, error) {
+	var err error
+	var js jetstream.JetStream
```

Comment:
Do we need to define those? Is it because of the `err` in L96 and L100?

Reply:
It is because of the combined "if"+"assignment"  on line 100. The `js` would be undefined later on line 105.